typescript/lib/constructor_calls.ts

```diff
@@ -0,0 +1,31 @@
+import * as bindingsJs from '../kurtosis_core_rpc_api_bindings/api_container_service_pb'; //TODO (comment) - I cannont merge with above line unfortunately, can only import one module at a time MAXIMUM
+import { ServiceID } from './services/service';
+
+
+export function newGetExecCommandArgs(serviceId: ServiceID, command: string[]): bindingsJs.ExecCommandArgs {
+    let result = new bindingsJs.ExecCommandArgs(); //TODO (comment) - this was also a pointer
```

Comment:
No need to fuss about pointers since Typescript doesn't have them. the BLUF is that I think Javascript, like Java, uses pointers under the hood for representing objects